webapi/src/main/scala/org/knora/webapi/core/AppServer.scala

```diff
@@ -80,10 +80,15 @@ final case class AppServer(
    *
    * @param requiresRepository If `true`, calls the AppRouter to populate the ontology caches, otherwise returns ()
    */
-  private def populateOntologyCaches(requiresRepository: Boolean): UIO[Unit] =
+  private def populateOntologyCaches(requiresRepository: Boolean): IO[Throwable, Unit] =
     for {
       _ <- state.set(AppState.LoadingOntologies)
-      _ <- ar.populateOntologyCaches.when(requiresRepository)
+      _ <-
+        ar.populateOntologyCaches
```

Comment:
What's the `ar`? :)

Reply:
the sound a pirate makes when he approves... obviously! :P  
Seriously, it stands for AppRouter, in this file there are a lot of those overly short variable names, and I got annoyed by it too:

```scala
final case class AppServer(
  state: State,
  ts: TriplestoreService,
  ru: RepositoryUpdater,
  as: ActorSystem,
  ar: AppRouter,
  iiifs: IIIFService,
  cs: CacheService,
  hs: HttpServer,
  appConfig: AppConfig
) 
```

But I considered changing this out of scope for my task at hand.

Reply:
Thanks for explanation argh!